common-api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -87,6 +87,7 @@ public static WorkbenchConfig createEmptyConfig() {
     public String projectId;
     public String shortName;
     public String oauthClientId;
+    public boolean alwaysTrace;
```

Comment:
Naming / docs nitpicks:

1) Please add a short comment describing the use of this flag (just enough so anyone on the team can quickly understand its usage / impact without further searching the codebase).

2) For some reason "alwaysTrace" sounds a bit misleading to me, as if there's a discrete time when we're "not tracing" when this flag is false. Maybe "traceAllRequests" is slightly clearer? Or a float-valued "traceSamplingRatio" to provide control over the sampling rate, though that's probably overkill.

Reply:
I can make it `traceAllRequests`. I think the sampling ratio is overkill, especially cause I couldn't find documentation on what the default is. Will add a comment.